### DISCUSSION TITLE
How to test in low version

### DISCUSSION THREAD

**Discussion by AIhang427** [Lightning-AI/pytorch-lightning#8103] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
There is no train.test () in the lower version. How to test the test data set?
my version: 0.4.6

**Comment by awaelchli** [comment, marked as answer]
Hi
PyTorch Lightning 0.4.6 is extremely old. You should consider upgrading. 
That being said, you can always test your model as you would in plain pytorch, because the LightningModule is also just a nn.Module:

```python
for inp, target in test_dataloader:
    pred = model(inp)
    test_loss = loss(pred, target)
    ...
```

**Reply by tchaton** [reply]
Hey @AIhang427,

I would recommend you to update to latest PyTorch Lightning version, lot of progress has been made since :)

Best,
T.C
